inference-engine/tests/unit/cpu/shape_inference_test.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include <openvino/core/coordinate_diff.hpp>
+#include <openvino/op/convolution.hpp>
+#include <openvino/op/parameter.hpp>
+#include <convolution_shape_inference.hpp>
+#include <openvino/op/ops.hpp>
+#include "utils/shape_inference/static_shape.hpp"
+
+using namespace ov;
+
+TEST(StaticShapeInferenceTest, ConvolutionTest) {
+    Strides strides{1, 1};
+    CoordinateDiff pads_begin{0, 0};
+    CoordinateDiff pads_end{0, 0};
+    Strides dilations{1, 1};
+    const auto auto_pad = op::PadType::SAME_LOWER;
+
+    auto data = std::make_shared<ov::op::v0::Parameter>(element::f32, PartialShape{-1, -1, -1, -1});
+    auto filters = std::make_shared<ov::op::v0::Parameter>(element::f32, PartialShape{-1, -1, -1, -1});
+
+    auto conv =
+            std::make_shared<op::v1::Convolution>(data, filters, strides, pads_begin, pads_end, dilations, auto_pad);
+
+    std::vector<PartialShape> input_shapes = {PartialShape{3, 6, 5, 5}, PartialShape{7, 6, 3, 3}}, output_shapes = {PartialShape{}};
+    shape_infer(conv.get(), input_shapes, output_shapes);
+
+    ASSERT_EQ(output_shapes[0], PartialShape({3, 7, 5, 5}));
+    ASSERT_EQ(conv->get_pads_begin(), (CoordinateDiff{1, 1}));
+    ASSERT_EQ(conv->get_pads_end(), (CoordinateDiff{1, 1}));
+
+    std::vector<StaticShape> static_input_shapes = {StaticShape{3, 6, 5, 5}, StaticShape{7, 6, 3, 3}}, static_output_shapes = {StaticShape{}};
+    shape_infer(conv.get(), static_input_shapes, static_output_shapes);
+
+    ASSERT_EQ(static_output_shapes[0], StaticShape({3, 7, 5, 5}));
+    ASSERT_EQ(conv->get_pads_begin(), (CoordinateDiff{1, 1}));
+    ASSERT_EQ(conv->get_pads_end(), (CoordinateDiff{1, 1}));
+}
+
+#if 0
+TEST(StaticShapeInferenceTest, ConvolutionTimeTest) {
```

Comment:
```suggestion
#if 0
TEST(StaticShapeInferenceTest, DISABLED_ConvolutionTimeTest) {
```
Why do we already have disabled test?

Reply:
yes, it prints to the cout, you see?